src/plugins/chart_expressions/expression_metric/public/components/metric_vis.tsx

```diff
@@ -307,6 +308,20 @@ export const MetricVis = ({
     pixelWidth = grid[0]?.length * maxTileSideLength;
   }
 
+  const [scrollChildHeight, setScrollChildHeight] = useState<string>('100%');
+  const scrollContainerRef = useRef<HTMLDivElement>(null);
+  const scrollDimensions = useResizeObserver(scrollContainerRef.current);
+
+  useEffect(() => {
+    const minTileHeight = 64; // TODO - magic number from the @elastic/charts side. would be nice to deduplicate
```

Comment:
@markov00 would it be feasible to somehow get this number from the charts library? Right now, it's pretty magic ✨

Reply:
sure, I can add that to the chart theme

Reply:
Great idea. Created an issue https://github.com/elastic/elastic-charts/issues/1788